etc/ramalama/registries.conf.d/000-shortnames.conf

```diff
@@ -0,0 +1,4 @@
+[aliases]
+  "granite" = "huggingface://instructlab/granite-7b-lab-GGUF/granite-7b-lab-Q4_K_M.gguf"
```

Comment:
Should add multiple
granite7b (and or granite-7b) ( I would love it if we stored these in cri-o by default, will talk about this next week)
granite-lab

Reply:
This file should also be in /usr/share.  Leave /etc for sysadmins. /usr/share for distributions.

Reply:
What ollama does... Which is kinda interesting is this format (like container images):

granite-code:8b
granite-code:3b
granite-code:latest (alias for 3b tag or whatever tag one selects)

Reply:
Yup that would make sense.

Reply:
At the moment I have:

"granite-lab-7b" and  "merlinite-lab-7b"

the ':' characters don't parse well in the .conf file with the python parser, but can try again to make ':' work if we really want the ':' character to be present.